testcases/kernel/syscalls/gethostid/gethostid01.c

```diff
@@ -160,6 +160,10 @@ int main(int ac, char **av)
 			tst_resm(TFAIL | TTERRNO, "gethostid failed");
 			continue;	/* next loop for MTKERNEL */
 		}
+		else {
+			tst_resm(TPASS, "gethostid is sucessfull & hostid = %d", TEST_RETURN);
+		}
```

Comment:
We need a good comment herem why we are disabling this code block, when shall we enable it back (AFter sycall implemented) TODO: #lsds/sgx-lkl/???

Reply:
updated the comment